libs/datasets/timeseries.py

```diff
@@ -1150,6 +1150,26 @@ def drop_regions_without_population(
     return mrts.get_locations_subset(locations_with_population)
 
 
+def _compute_derived_vaccinations_initiated(dataset: MultiRegionDataset):
+    administered = dataset.timeseries[CommonFields.VACCINES_ADMINISTERED]
+    completed = dataset.timeseries[CommonFields.VACCINATIONS_COMPLETED]
+    derived_initiated = administered - completed
+    initiated = dataset.timeseries[CommonFields.VACCINATIONS_INITIATED]
+
+    return initiated.fillna(derived_initiated)
```

Comment:
I only want to fill for regions that have no vaccinations_initiated data at all. what's the best way to do this?